src/FridgeSecondary.java

```diff
@@ -0,0 +1,69 @@
+import components.map.Map;
+import components.map.Map1L;
+
+public abstract class FridgeSecondary
+        implements FridgeKernel, FridgeSecondaryInterface {
+    private Map<String, Integer> fridgeItems;
+    private Map<String, Integer> freezerItems;
+
+    public FridgeSecondary() {
```

Comment:
No fields or constructors in the abstract class. Use the kernel methods directly to get what you need.